baseplate/experiments/providers/__init__.py

```diff
@@ -54,16 +60,42 @@ def parse_experiment(config):
     assert isinstance(experiment_id, int)
     name = config["name"]
     owner = config.get("owner")
-    experiment_config = config["experiment"]
-    expiration = datetime.strptime(config["expires"], ISO_DATE_FMT).date()
+    if "start_ts" in config and "stop_ts" in config:
+        start_ts = config["start_ts"]
+        stop_ts = config["stop_ts"]
+    elif "expires" in config:
+        warn_deprecated(
+            "The 'expires' field is in experiment %s deprecated, you should "
+            "use 'start_ts' and 'stop_ts'." % name
+        )
+        start_ts = time.time()
+        expires = datetime.strptime(config["expires"], ISO_DATE_FMT)
+        epoch = datetime(1970, 1, 1)
```

Comment:
does this need to be adjusted for time offset?

Reply:
No, they're both timezone naive:

```python
In [1]: import pytz

In [2]: from datetime import datetime

In [3]: ISO_DATE_FMT = "%Y-%m-%d"

In [4]: DATE_EXPIRES = "2100-01-01"

In [5]: expires = datetime.strptime(DATE_EXPIRES, ISO_DATE_FMT)

In [6]: epoch = datetime(year=1970, day=1, month=1)

In [8]: expires_utc = expires.replace(tzinfo=pytz.utc)

In [12]: epoch_utc = epoch.replace(tzinfo=pytz.utc)

In [13]: (expires_utc - epoch_utc).total_seconds()
Out[13]: 4102444800.0

In [14]: (expires - epoch).total_seconds()
Out[14]: 4102444800.0

In [15]: (expires_utc - epoch_utc).total_seconds() == (expires - epoch).total_seconds()
Out[15]: True
```